platform/platform-modern/src/main/java/tc/oc/pgm/platform/modern/listeners/ModernListener.java

```diff
@@ -56,6 +59,33 @@ public void onPlayerAttackEntity(PrePlayerAttackEntityEvent event) {
     handleCall(pgmEvent, event);
   }
 
+  @EventHandler(ignoreCancelled = true)
+  public void onGlideToggle(EntityToggleGlideEvent event) {
+    if (event.getEntity() instanceof Player player) {
+      PlayerGlideToggleEvent pgmEvent = new PlayerGlideToggleEvent(player, player.isGliding());
+      Bukkit.getScheduler().runTask(PGM.get(), () -> handleCall(pgmEvent, event));
+    }
+  }
+
+  @EventHandler(ignoreCancelled = true)
+  public void onRiptideStart(org.bukkit.event.player.PlayerRiptideEvent event) {
+    Player player = event.getPlayer();
+
+    Bukkit.getScheduler()
```

Comment:
pgm has its own executor under `PGM.get().getExecutor()` that allows scheduling and cancelling futures.
You can see an example in `MatchTabManager`'s `RenderTask`, where it schedules itself as a future in the constructor then the run method can cancel that future, or in `MapDevCommand`'s `DisplayRunner` which also creates the future and cancels it itself in the run method